### DISCUSSION TITLE
Welcome, SAN.D!

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#664] [Subscribe]
I would love to contribute to this repo...can I get Permission?

**Comment by ParanoidUser** [comment]
Ahoy, @SANKARAMDAS!

Thank you for your interest and participation in this project! Do you have any specific things in mind that you would like to work on?

Here are just a few of the areas where help is most needed:
 - We're always on the lookout for more exciting questions & puzzles to share with the community;
 - Create, improve, and maintain badge earning guides and materials;
 - Increase user engagement and attract new users interested in the project;

**Reply by SANKARAMDAS** [reply]
Sure @ParanoidUser, I would love to participate and do my best to improve where it needs to be improved.

**Reply by ParanoidUser** [reply]
The invitation to join the collaborator group has been sent to you. Welcome aboard! 😊
